.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - "2.7"
```

Comment:
We would like to have python 3. and not Python 2.

Reply:
we can change it in future, I need to see that it works well and I know that it woks with python2 now (it is connected with installation via setup.py actually it is done under python2). You can change it then if you know how. or better to say, install it to both of them python2.*/site-packages and python3.*/site-packages